src/App.css

```diff
@@ -60,75 +61,99 @@
 
 
 /* Remove Body Scroll */
-
 ::-webkit-scrollbar {
     width: 0px;
     /* remove scrollbar space */
     background: transparent;
     /* optional: just make scrollbar invisible */
 }
-
-@-moz-document url-prefix() {
-    html,
-    body {
-        overflow: hidden !important;
-    }
-}
-
-
 /* Remove Body Text Highlight */
 
 ::selection {
     color: none;
     background: none;
 }
-
-
 /* For Mozilla Firefox */
 
 ::-moz-selection {
     color: none;
     background: none;
 }
 
+
+@-moz-document url-prefix() {
+    html,
+    body {
+        overflow: hidden !important;
+    }
+}
+html,
 body {
     background-color: #282c34;
-    min-height: 100vh;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
-    font-size: calc(10px + 2vmin);
-    color: white;
+    color: #fff;
+    font-family: var(--main-font);
 }
-
-
-/* Check over after merge */
-
-.App {
-    text-align: center;
+h1, h2, h3, p, a, div, input{
+    color: #fff!important; 
+    font-family: var(--main-font);
+}
+ 
+#root {
+    height: 100vh; /* Use vh as a fallback for browsers that do not support Custom Properties */
+    height: calc(var(--vh, 1vh) * 100);
+    margin: 0 auto;
+    max-width: 100%;
 }
 
-.App-logo {
-    height: 40vmin;
-    pointer-events: none;
+.pageWrap {
+    height: -webkit-fill-available;
+    height: -moz-available;
+    height: fill-available;
+    padding: 15px;
+    text-align: center;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-    .App-logo {
-        animation: App-logo-spin infinite 20s linear;
+/* When the browser is at least 600px and above */
+@media screen and (min-width: 1000px) {
+    #root {
+        max-width: 50%;
     }
-}
+  }
 
-.App-link {
-    color: #61dafb;
+#pageBody {
+    padding: 15px;
+    background-color: #1d2025;
+    /*
```

Comment:
No reason to leave commented-out code here

Reply:
Removed comment code.